siem_integrations/clx_query/bin/clx_query.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
```

Comment:
This should be 2020, not 2019. Same for the other headers you added.

Reply:
sorry, my bad. I have updated the files.